crypto/homo/cl/cl.go

```diff
@@ -50,6 +53,8 @@ var (
 	ErrFailedVerify = errors.New("failed verify")
 	//ErrFailedGenerateG is returned if g is the identity element
 	ErrFailedGenerateG = errors.New("failed generate non-identity g")
+	//ErrNotOddPrime is returned if p is not an odd prime
+	ErrNotOddPrime = errors.New("the message space p is not an odd prime")
```

Comment:
errors.New("the message space p is not an odd prime") -> errors.New("not an odd prime")

Reply:
done